htpasswd/auth_test.go

```diff
@@ -20,7 +21,7 @@ bob-foo:{fooo}sdcsdcsdc/BfQ=
 `
 
 func TestAuth_Hashes(t *testing.T) {
-	auth, err := NewAuth(writeTmpfile(testfile))
+	auth, err := NewAuth([]string{writeTmpfile(testfile)})
```

Comment:
Please change the signature of  

    func writeTmpfile(filename string) string

to 

    func writeTmpfiles(filename ...string) []string

This makes the test more readable.